vue/src/mapstyle/rdwatchtiles.ts

```diff
@@ -96,14 +103,38 @@ export const buildSiteFilter = (
 
 const rdwatchtiles = "rdwatchtiles";
 const urlRoot = `${location.protocol}//${location.host}`;
-const observationColor: DataDrivenPropertyValueSpecification<string> = [
+const annotationColor: DataDrivenPropertyValueSpecification<string> = [
   "case",
   ["==", ["get", "label"], 1],
   "#1F77B4",
   ["==", ["get", "label"], 2],
   "#A020F0",
   ["==", ["get", "label"], 3],
   "#2CA02C",
+  ["==", ["get", "label"], 4],
+  "#2f4f4f",
+  ["==", ["get", "label"], 5],
+  "#228b22",
+  ["==", ["get", "label"], 6],
+  "#7f0000",
+  ["==", ["get", "label"], 7],
+  "#00008b",
+  ["==", ["get", "label"], 8],
+  "#ff8c00",
+  ["==", ["get", "label"], 9],
+  "#ffff00",
+  ["==", ["get", "label"], 10],
+  "#1e90ff",
+  ["==", ["get", "label"], 11],
+  "#00ffff",
+  ["==", ["get", "label"], 12],
+  "#ff00ff",
+  ["==", ["get", "label"], 13],
+  "#00ff00",
+  ["==", ["get", "label"], 14],
+  "#ff69b4",
+  ["==", ["get", "label"], 15],
+  "#ffe4c4",
```

Comment:
Not needed for this PR but in the future we should probably build a typescript Type definition for these annotations/colors/labels.  I didn't know this before but probably basing it off of the lookups.json would work. Then we have a function which could generate this and the below text-labels.  It would be nice to have a direct tying of the color to the text displayed.

Reply:
Agreed!